packages/dawai-microservice/src/decorators/crud.decorator.ts

```
@@ -0,0 +1,13 @@
+import { CrudDecoratorOptions } from '../decorator.options';
+import { metadataStorage } from './metadata.storage';
+
+export function crud(options: CrudDecoratorOptions): MethodDecorator {
+  return function<T> (
+    target: any, // Reverted to 'any' to see if it resolves signature mismatch
+    propertyKey: string | symbol,
+    descriptor: TypedPropertyDescriptor<T>
+  ): TypedPropertyDescriptor<T> | void {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)